.github/workflows/stale-pr.yml

```diff
@@ -7,11 +7,12 @@ jobs:
   stale:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/stale@v3
+      - uses: actions/stale@v9
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          stale-pr-message: 'This PR was marked stale due to lack of activity. It will be closed in 7 days.'
+          stale-pr-message: 'This PR was marked stale due to lack of activity. It will be closed in 7 days. Consider applying the label `work in progress` to avoid this being marked as stale again.'
```

Comment:
Non-approvers won't be able to add this label - which makes sense but the message might therefore confuse PR authors.

Reply:
True.. I removed it for now.